### DISCUSSION TITLE
A quicker and more ephemeral communication channel?

### DISCUSSION THREAD

**Discussion by rozsasarpi** [BAMresearch/probeye#47] [Ideas]
I would like to have a quick communication channel to/with those who are involved in this repo and related activities, where questions that will become outdated quickly yet relevant at the moment could be asked quickly without opening an issue, sending an email, etc. These questions and discussions are not worth being systematically archived.

Basically a chat like application like WhatsApp. Within TNO I can use Teams for this but that has organizational boundaries. I see that many projects use gitter for this purpose, which could be tried.

Please let me know what you think.

Is there an `@all` in GitHub to notify everyone? Until I figure it out, I just tag users: @joergfunger @aklawonn @TTitscher @JanKoune @atulag0711

**Comment by atulag0711** [comment]
Never used gitter, but slack can also be a good option IMO.
